.github/workflows/ci.yml

```diff
@@ -26,7 +26,6 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: 18
-          cache: 'yarn'
```

Comment:
It worked.
I found out that we've reached the maximum limit for total cache storage (10.2 GB of 10 GB Used)

<img width="654" alt="image" src="https://github.com/vtex/faststore/assets/3356699/f98e40b6-c2c1-4d1b-b672-afb46f271374">